pysrc/utils/sampling_methods.py

```diff
@@ -65,17 +65,16 @@ def shortest_value_sample(data: list[list[str]],
 
 def all_distinct_sample(data: list[list[str]],
                  num_samples: int,
-                 num_entries: int) -> list[str]:
+                 num_entries: int) -> pd.Series:
     tmp = pd.Series(data)
     df = tmp.loc[tmp.astype(str).drop_duplicates().index]
 
 
     return df.iloc[:num_samples]
 
-#TODO Return Type is messed up
 def evenly_spaced_sample(data: list[list[str]],
                          num_samples: int,
-                         num_entries: int) -> list[str]:
+                         num_entries: int) -> pd.Series:
```

Comment:
Also the type of the list below has to be updated.

Reply:
@yjojo17 The type of `sampling_methods_dict` has to be updated. It represents a method returning `list[str]` which was correct, but now it should return `pd.Series`.